extensions/cornerstone/src/utils/measurementServiceMappings/LivewireContour.ts

```diff
@@ -142,7 +142,7 @@ function getDisplayText(annotation, displaySet, customizationService) {
   const frameText = displaySet.isMultiFrame ? ` F: ${frameNumber}` : '';
 
   const { SeriesNumber } = displaySet;
-  if (SeriesNumber) {
+  if (SeriesNumber !== undefined) {
```

Comment:
How about

```
displayText.push(`S: ${SeriesNumber ?? ''}${instanceText}${frameText}`);
```

Reply:
i like that, thats good

Reply:
actually I tried it, doesn't work too, maybe if its 0 its also a problem